packages/expo-ui/src/jetpack-compose/TextInput/index.tsx

```diff
@@ -12,7 +12,7 @@ export type TextInputProps = {
   /**
    * Additional styles to apply to the TextInput.
    */
-  style?: StyleProp<ViewStyle>;
+  style?: Pick<TextStyle, 'color' | 'fontSize' | 'fontWeight' | 'fontFamily'>;
```

Comment:
doesn't this drop support for styling text width/height? I think this works in compose (?)